lib/members_page.rb

```diff
@@ -0,0 +1,9 @@
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  field :members do
+    noko.css('ul.lisitng_resultat li').map do |li|
+      MemberSection.new(response: Scraped::Request.new(url: url).response, noko: li)
```

Comment:
why are you generating a new response here rather than pass the existing one through?